apps/storybook/stories/components/radio/Radio.stories.tsx

```diff
@@ -40,7 +40,7 @@ export default meta;
 type Story = StoryObj<typeof KvibRadio>;
 
 export const Radio: Story = {
-  args: { children: "Klikk her", colorScheme: "green" },
```

Comment:
Jeg ville beholdt denne linja da man kan se at grønn er huket av for her. Nå virker det ikke som noen av de er valgt, når grønn er det.
![Screenshot 2023-06-07 at 14 42 59](https://github.com/kartverket/kvib/assets/22708819/ca1c91c0-6342-489a-a88d-e859745a8654)